### PR TITLE
HWCC-81 Added feature description and Removed unused library

### PR DESCRIPTION
**Pull request by ravipwaghmare**
PR is for issues 
https://github.com/metal3-io/hardware-classification-controller/issues/81

Kindly review
@Ashughorla  @digambar15  @maelk

### CODE REVIEW

**Comment by Ashughorla** [comment]
Changes looks good to me

File: go.mod [hunk 1]
```diff
@@ -4,15 +4,15 @@ go 1.16
 
 require (
 	github.com/go-logr/logr v0.2.1
+	github.com/mattn/go-isatty v0.0.12 // indirect
```
Comment (digambar15) [inline go.mod:7]: Is this lib approved by CNCF ?

**Review by digambar15 — CHANGES_REQUESTED** [review]
I would like to understand reason behind this lib. If there are changed pushed using the lib, we may need to revert those as well.

File: go.mod (hunk 1 above)
Comment (ravipwaghmare) [reply go.mod:7]: @digambar15  This is the indirect library that is not used in HWCC.
But used by controller-gen from sigs.k8s.io/controller-tools which is Kubernetes library.
This is the flow of the library.

- github.com/metal3-io/hardware-classification-controller
-- sigs.k8s.io/controller-tools/cmd/controller-gen
--- sigs.k8s.io/controller-tools/pkg/genall/help/pretty
---- github.com/fatih/color

Comment (digambar15) [reply go.mod:7]: okay. Looks good.

**Review by digambar15 — APPROVED** [review]
Looks Good

**Pull request merged**